docs/kazu_webservice.rst

```diff
@@ -1,2 +1,70 @@
-TBA
-===
+Kazu as a Web Server (REST API) with FastAPI
+============================================
+
+The Kazu framework can be run in REST API mode with a few additional steps. This page provides a simple tutorial on how to host a REST API server using FastAPI.
+
+Installation
+------------
+
+First, clone the repository:
+
+.. code-block:: bash
+
+    git clone https://github.com/AstraZeneca/KAZU.git
```

Comment:
rather than cloning the repo, perhaps it's better to install from pip, as the pip version  is tested with the model pack version from the releases page? (main branch may have breaking changes with previous model pack release)

Reply:
Sounds great! I just tested with my Mac OS environment and installation works well.